src/pages/MyLists/__tests__/component.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import MyLists from '../component';
+
+test('MyLists component matches snapshot', () => {
```

Comment:
Wrap all in `describe` block and use `it` for test

Reply:
Done